_inc/client/components/footer/index.jsx

```diff
@@ -91,7 +91,7 @@ export const Footer = React.createClass( {
 		return (
 			<div className={ classes }>
 				<div className="jp-footer__a8c-attr-container">
-					<a href="http://automattic.com" target="_blank">
+					<a href="https://automattic.com" target="_blank">
 					<svg className="jp-footer__a8c-attr" x="0" y="0" viewBox="0 0 935 38.2" enable-background="new 0 0 935 38.2">
```

Comment:
Unrelated to this changeset, but the indentation is a bit off. Down a few lines we also do `</svg></a>` on the same line. Just making a note.